lib/utils.rb

```diff
@@ -25,7 +25,7 @@ def self.normalize_uri(uri)
     begin
       URI(uri)
     rescue URI::Error
-      URI(uri.to_s.gsub(/[^\-_.!~*'()a-zA-Z\d;\/?:@&=+$,\[\]]+/) { |unsafe|
+      URI(uri.to_s.gsub(/[^\-_.!~*'()a-zA-Z\d;\/?:@&=+$,]+/) { |unsafe|
```

Comment:
This will break a URL with an IPv6 address host, as pointed out by [this comment](http://stackoverflow.com/a/35460502/5342386).